src/cloud/components/settings/MembersTab.tsx

```diff
@@ -338,16 +336,7 @@ const MembersTab = () => {
 
```

Comment:
Remove unused `tab, setTab` state variables
dont forget to run `npm run tsc` in dev to check if you do not have issues

Reply:
Removed `tab, setTab` and no problem found with `npm run tsc`